.github/workflows/generate-docs.yaml

```diff
@@ -0,0 +1,54 @@
+name: Generate Documentation
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Generate documentation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+      - uses: actions/setup-go@v1
+        with:
+          go-version: '1.16'
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.9
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: docs-pip-${{ runner.os }}-${{ hashFiles('doc-requirements.txt') }}-${{ hashFiles('doc-requirements.in') }}
+          restore-keys: docs-pip-
+      - name: Install Dependencies
+        run: |
+          # Install all requirments
+          make doc-requirements.txt
+      - name: Generate documentation
+        run: |
+          make -C docs gendocs
```

Comment:
It is failing because the dev tool is not installed. Like `sphinx-build`. Any idea why?

Reply:
Can we bring in the doc requirements from flytesnacks , or better can that doc-requirements be moved to boilerplate and be used in all repos . So that all the repo where we generate docs have these dependencies pulled in and built

Reply:
that means we will copy these docs requirement and Make the command from the boilerplate. Right?

Reply:
If yes then do it as a seprate pr